Libraries/LibCrypto/CMakeLists.txt

```diff
@@ -28,14 +28,13 @@ set(SOURCES
     Curves/Ed25519.cpp
     Curves/X25519.cpp
     Curves/X448.cpp
-    Hash/BLAKE2b.cpp
-    Hash/MD5.cpp
-    Hash/SHA1.cpp
-    Hash/SHA2.cpp
```

Comment:
I believe these removals should be in the last commit.

Reply:
Yep, fixed